plugins/module_utils/virtual_disk.py

```diff
@@ -43,21 +45,20 @@ def from_ansible(cls, ansible_data: TypedVirtualDiskFromAnsible) -> VirtualDisk:
 
     @classmethod
     def from_hypercore(cls, hypercore_data: Dict[Any, Any]) -> VirtualDisk:
-        return cls(
-            name=hypercore_data["name"],
-            uuid=hypercore_data["uuid"],
-            block_size=hypercore_data["blockSize"],
-            size=hypercore_data["capacityBytes"],
-            # allocated_size=hypercore_data["totalAllocationBytes"],
-            replication_factor=hypercore_data["replicationFactor"],
-        )
+        try:
+            return cls(
+                name=hypercore_data["name"],
+                uuid=hypercore_data["uuid"],
+                block_size=hypercore_data["blockSize"],
+                size=hypercore_data["capacityBytes"],
+                # allocated_size=hypercore_data["totalAllocationBytes"],
+                replication_factor=hypercore_data["replicationFactor"],
+            )
+        except KeyError as e:
+            raise errors.MissingValueHypercore(e)
```

Comment:
It can potentially happen if virtual disk is missing parameter data (corrupted data on cluster) and is then returned with missing keys. 
We had some examples in the past broken with VMs.
This is just to give a better error LOG when fixing bugs.
@just
Do you think there is a potential downside to this?